sqlglot/dialects/prql.py

```diff
@@ -78,6 +79,11 @@ def _parse_selection(self, query: exp.Query, append: bool = True) -> exp.Query:
 
             return query.select(*selects, append=append, copy=False)
 
+        def _parse_take(self, query: exp.Query) -> exp.Query:
+            if self._match(TokenType.NUMBER):
+                val = self._prev.text  # TAKE <NUMBER>
+            return query.limit(int(val))  # TODO: TAKE for ranges a..b
```

Comment:
Let's simplify this to:

```suggestion
            return query.limit(self._parse_number())  # TODO: TAKE for ranges a..b
```

Reply:
thanks George!

Reply:
Sure thing! I guess we need to update the type hint so it's optional as well.